Dockerfile

```diff
@@ -13,4 +13,7 @@ RUN echo "/home/octave/bioformats_package.jar" >> /home/octave/javaclasspath.txt
 
 
 RUN echo "pkg install bioformats-octave-$VERSION.tar.gz" | octave
+
+RUN wget https://downloads.openmicroscopy.org/images/OME-TIFF/2016-06/tubhiswt-3D.zip
```

Comment:
Absolutely. Using `utm_source` or `--user-agent` as we do in `omego` would allow us to filter out these from the stats? Happy to go with `utm_source` if that's the best recommendation for now.